packages/theme/src/ee/store/dashboard/roles.ts

```diff
@@ -40,7 +40,7 @@ export const useDashboardRoles = defineStore("dashboardRoles", () => {
     const roleIdx = roles.value.findIndex((item) => item.id === role.id);
     if (roleIdx === -1) return;
 
-    Object.assign(roles.value[roleIdx], role);
+    if (roles.value[roleIdx]) Object.assign(roles.value[roleIdx], role);
```

Comment:
Same as [this](https://github.com/logchimp/logchimp/pull/1447/files#r2506840990)